src/kibana/services/es.js

```diff
@@ -12,6 +12,7 @@ define(function (require) {
         host: configFile.elasticsearch,
         log: 'info',
         requestTimeout: 0,
+        apiVersion: '1.4',
```

Comment:
It would probably be good to put this in `server/lib/elasticsearch_client.js` too, no?